blocks/accordion/accordion.css

```diff
@@ -0,0 +1,79 @@
+.accordion .accordion-item {
+  margin: 16px 20px;
+  padding: 18px 54px;
+  border-radius: 10px;
+  border-color: rgb(229 231 235);
+  border-width: 0;
+  box-shadow: 0 3px 8px 0 rgb(0 0 0 / 10%);
+  display: flex;
+  flex-direction: column;
+}
+
+@media (min-width:992px) {
```

Comment:
I noticed, that when opening the question and decreasing the window size the height or overflow does not adjust correctly: 

https://github.com/hlxsites/bitdefender/assets/30388999/789b2d69-186f-4826-a839-fc8751ffff6d

Reply:
Good catch @johan-t ! :-)